cfgov/v1/preprocessed/css/cf-enhancements.less

```diff
@@ -1245,8 +1275,31 @@ tbody {
     &__border-right {
         border-right: 1px solid @block__border;
     }
+
+    &__flush {
+        margin-top: 0 !important;
+        margin-bottom: 0 !important;
+        margin-right: -(@grid_gutter-width / 2);
+        margin-left: -(@grid_gutter-width / 2);
+        .respond-to-min(600px, {
+            margin-right: -@grid_gutter-width;
+            margin-left: -@grid_gutter-width;
+        });
+    }
+
+    &__bg {
+        padding: unit(@grid_gutter-width / @base-font-size-px, em)
+                 unit((@grid_gutter-width / 2) / @base-font-size-px, em);
```

Comment:
To make this easier to migrate to CF I'm just taking what's [currently in CF](https://github.com/cfpb/cf-layout/blob/gh-pages/src/cf-layout.less#L1268-L1276) and augmenting it, so that we can just copy and paste the whole block over and know it'll work.

Does that answer what you're getting at?